.circleci/config.yml

```diff
@@ -293,7 +293,7 @@ commands:
             cd rust
 
             TARBALL_PATH="/tmp/${CIRCLE_PROJECT_REPONAME}-$(uname)-optimized.tar.gz"
-            RUSTFLAGS="-C target-feature=$(cat rustc-target-features-optimized.json | jq -r '.[].rustc_target_feature' | tr '\n' ',')"
+            RUSTFLAGS="-C target-cpu=native -C target-feature=$(cat rustc-target-features-optimized.json | jq -r '.[].rustc_target_feature' | tr '\n' ',')"
```

Comment:
@cryptonemo I don't think this is a good idea, this will specialize to the CPU that circle ci has, ending up in randomly hard to detect issues. If folks want to build against native, they will need to build for themselves.

Reply:
These builds aren't published. Maybe better to remove the CI jobs?

Reply:
ah right, I always get confused with this CI job